### ISSUE TITLE
Bot stuck online after master logout

### ISSUE THREAD

**Issue by Atsuumi** [liyunfan1223/mod-playerbots#74]
Bots are still stuck online, even if their master is offline for hours. Sometimes, if you kick or reset the bot stats, it can cause a crash

- Insta logout is active

**Comment by evrydayjunglist** [comment]
Did you use the addclass command to add your bots, or are they random bots that were already in the world?

**Comment by Atsuumi** [comment]
> Did you use the addclass command to add your bots, or are they random bots that were already in the world?

Alt bots added with playerbot bot add command

**Comment by evrydayjunglist** [comment]
I don't experience what you're describing. Do you have the latest mod-playerbots and the latest azerothcore playerbot branch (the AC playerbot branch was just recently updated with commits outside of the core)?

**Comment by evrydayjunglist** [comment]
I'm not using alts though. I'll try that.

**Comment by Atsuumi** [comment]
Yes, everything up to date, verified multiple times. I guess it happens on an unstable network ( high ping ) maybe the packets are not being sent corectly and the bot remains online, but seen it happens on a stable network too, but rare

**Comment by Dreathean** [comment]
@Atsuumi Are you still experiencing this issue? I haven't had it happen for me with altbots
